pkg/apis/apps/v1alpha1/yurtappdaemon_types.go

```diff
@@ -108,7 +108,7 @@ type YurtAppDaemonCondition struct {
 // +k8s:openapi-gen=true
```

Comment:
pkg/apis/apps/v1alpha1/yurtappdaemon_types.go file can be removed.

Reply:
done